File: Beginner/Mahasena/Mahasena.js
const readline = require('prompt-sync')({sigint: true});
const n = Number(readline());
let arr = new Array(n);
let num = readline().split(" ");
for (let i=0; i<n; i++) {
    arr[i] = num[i];
}
let even=0, odd=0;
for (let j=0; j<n; j++) {
    if (arr[j] % 2 == 0) {
        even++;
    } else {
        odd++;
    }
}
if (even > odd) {
    console.log("READY FOR BATTLE");
} else {
    console.log("NOT READY");
}

File: Beginner/Mahasena/Mahasena.py
# cook your dish here
n = int(input())
lst = list(map(int, input().split()))
even, odd = 0, 0
for i in range(n):
    if lst[i] % 2 == 0:
        even += 1
    else:
        odd += 1
print("READY FOR BATTLE" if even > odd else "NOT READY")
